dlp/api/Snippets.Tests/DeidentifyTableWithRowSuppressTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class DeidentifyTableWithRowSuppressTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public DeidentifyTableWithRowSuppressTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestDeidentifyTable()
+        {
+            var response = DeidentifyTableWithRowSuppress.DeidentifyTable(_fixture.ProjectId);
+            Assert.Equal(2, response.Rows.Count);
```

Comment:
Please also assert that response doesn't contain 101 (age on suppressed row)

Reply:
Added.